Formula/s/stellar-core.rb

```diff
@@ -63,9 +63,7 @@ def install
       accountsubentriescount
       bucketlistconsistent
       topology
-      upgrades
```

Comment:
I have removed the upgrades test because one of the tests fails on macOS 13 x64 only due to what appears to be an output formatting issue. See this issue:
- https://github.com/stellar/stellar-core/issues/4214.

The test categories that have been selected to run after install appear to be arbitrary. There are 120 test categories in total (according to `stellar-core test -t`). I don't see much reason to select the above four over others that are available. It's possible this small subset were selected to get some signal that the binary works. If that's the case, I don't think it's a concern to remove upgrades.

We could also consider replacing this test with a much simpler test that checks that the binary reports a specific version.

Reply:
> We could also consider replacing this test with a much simpler test that checks that the binary reports a specific version.

In my experience those rarely show problems even when problems exist. But I think removing this particular test from the scope is fine.